# <font style = "color:#32a852; font-size:36px"><u>CASE STUDY - 4</u> Athul P J</font>

In [1]:
import pandas as pd
import scipy.stats as sp

In [2]:
df = pd.read_csv("Sales_add.csv")
df.head(10)

Month      Region      Manager  Sales_before_digital_add(in $)  \
0   Month-1  Region - A  Manager - A                          132921   
1   Month-2  Region - A  Manager - C                          149559   
2   Month-3  Region - B  Manager - A                          146278   
3   Month-4  Region - B  Manager - B                          152167   
4   Month-5  Region - C  Manager - B                          159525   
5   Month-6  Region - A  Manager - B                          137163   
6   Month-7  Region - C  Manager - C                          130625   
7   Month-8  Region - A  Manager - A                          131140   
8   Month-9  Region - B  Manager - C                          171259   
9  Month-10  Region - C  Manager - B                          141956   

   Sales_After_digital_add(in $)  
0                         270390  
1                         223334  
2                         244243  
3                         231808  
4                         258402  
5                         256948  
6                         222106  
7                         230637  
8                         226261  
9                         193735

In [4]:
df.isnull().sum()

Month                             0
Region                            0
Manager                           0
Sales_before_digital_add(in $)    0
Sales_After_digital_add(in $)     0
dtype: int64

# <font style = "color:#ff5e00; font-size:24px">H0: There is no increase in sales after stepping into Digital Marketing <br> H1: There is increase in sales after stepping into Digital Markrting </font>

In [15]:
t1 = sp.ttest_rel(df["Sales_before_digital_add(in $)"],df["Sales_After_digital_add(in $)"],alternative = "less")

In [16]:
print(t1)

Ttest_relResult(statistic=-12.09070525287017, pvalue=3.168333502287889e-11)


In [18]:
print(df["Sales_before_digital_add(in $)"].describe())
print(df["Sales_After_digital_add(in $)"].describe())

count        22.000000
mean     149239.954545
std       14844.042921
min      130263.000000
25%      138087.750000
50%      147444.000000
75%      157627.500000
max      178939.000000
Name: Sales_before_digital_add(in $), dtype: float64
count        22.000000
mean     231123.727273
std       25556.777061
min      187305.000000
25%      214960.750000
50%      229986.500000
75%      250909.000000
max      276279.000000
Name: Sales_After_digital_add(in $), dtype: float64


# <font style = "color:#ff5e00; font-size:24px">The pvalue is 3.168333502287889*e^-11 which is less than 0.05 we can reject the null hypothesis and conclude that the sales increased after stepping into digital marketing</font>

In [ ]:
#check whether there is any dependency between thefeatures “Region” and “Manager”

# <font style = "color:#ff0051; font-size:24px">H0: There is no dependency between thefeatures “Region” and “Manager” <br> H1: There is dependency between thefeatures “Region” and “Manager” </font>

In [32]:
crosstab = pd.crosstab(df["Region"], df["Manager"])
crosstab

Manager     Manager - A  Manager - B  Manager - C
Region                                           
Region - A            4            3            3
Region - B            4            1            2
Region - C            1            3            1

In [33]:
sp.chi2_contingency(crosstab)

(3.050566893424036,
 0.5493991051158094,
 4,
 array([[4.09090909, 3.18181818, 2.72727273],
        [2.86363636, 2.22727273, 1.90909091],
        [2.04545455, 1.59090909, 1.36363636]]))

# <font style = "color:#ff0051; font-size:24px">Since pvalue we got( 0.549 ) was higher than 0.05. So we can't reject the null hypothesis H0, so we can conclude that there is no dependency between thefeatures “Region” and “Manager” </font>